app/src/main/java/com/github/geohunt/app/ui/theme/Color.kt

```diff
@@ -2,7 +2,25 @@ package com.github.geohunt.app.ui.theme
 
 import androidx.compose.ui.graphics.Color
 
-val Purple200 = Color(0xFFBB86FC)
-val Purple500 = Color(0xFF6200EE)
-val Purple700 = Color(0xFF3700B3)
-val Teal200 = Color(0xFF03DAC5)
\ No newline at end of file
+val md_theme_light_primary = Color(0xFFA53C00)
+val md_theme_light_onPrimary = Color(0xFFFFFFFF)
+val md_theme_light_secondary = Color(0xFF99461E)
+val md_theme_light_onSecondary = Color(0xFFFFFFFF)
+val md_theme_light_error = Color(0xFFBA1A1A)
+val md_theme_light_onError = Color(0xFFFFFFFF)
+val md_theme_light_background = Color(0xFFFFFBFF)
+val md_theme_light_onBackground = Color(0xFF201A18)
+val md_theme_light_surface = Color(0xFFFFFBFF)
+val md_theme_light_onSurface = Color(0xFF201A18)
+val md_theme_dark_primary = Color(0xFFFFB598)
+val md_theme_dark_onPrimary = Color(0xFF591D00)
+val md_theme_dark_secondary = Color(0xFFFFB597)
+val md_theme_dark_onSecondary = Color(0xFF591D00)
+val md_theme_dark_error = Color(0xFFFFB4AB)
+val md_theme_dark_onError = Color(0xFF690005)
+val md_theme_dark_background = Color(0xFF201A18)
+val md_theme_dark_onBackground = Color(0xFFEDE0DC)
+val md_theme_dark_surface = Color(0xFF201A18)
+val md_theme_dark_onSurface = Color(0xFFD0C4C0)
```

Comment:
Should you not remove all of this because already defined as a resource (which by the way is better)

Reply:
no, it is required by the theme (idk why, that's how the theme came out when generated)